apps/ledger-live-mobile/src/families/ethereum/EditTransaction/TransactionAlreadyValidatedError.tsx

```diff
@@ -0,0 +1,138 @@
+import React, { memo } from "react";
+import { View, StyleSheet } from "react-native";
+import { Trans } from "react-i18next";
+import { useTheme } from "styled-components/native";
+import { BoxedIcon, Flex, Text } from "@ledgerhq/native-ui";
+import { CloseMedium } from "@ledgerhq/native-ui/assets/icons";
+import { CompositeScreenProps } from "@react-navigation/core";
+
+import TranslatedError from "../../../components/TranslatedError";
+import SupportLinkError from "../../../components/SupportLinkError";
+import Button from "../../../components/Button";
+import NeedHelp from "../../../components/NeedHelp";
+import {
+  StackNavigatorNavigation,
+  StackNavigatorProps,
+} from "../../../components/RootNavigator/types/helpers";
+import { BaseNavigatorStackParamList } from "../../../components/RootNavigator/types/BaseNavigator";
+import { EditTransactionParamList } from "../editTransactionNavigatorParamsList";
+import { ScreenName } from "../../../const";
+
+type Props = CompositeScreenProps<
+  StackNavigatorProps<
+    EditTransactionParamList,
+    ScreenName.TransactionAlreadyValidatedError
+  >,
+  StackNavigatorProps<BaseNavigatorStackParamList>
+>;
+
+const TransactionAlreadyValidatedErrorComponent = ({
+  navigation,
+  route,
+}: Props) => {
+  const { colors } = useTheme();
+  const { error } = route.params;
+
+  return (
+    <View
+      style={[
+        styles.root,
+        {
+          backgroundColor: colors.background.main,
+        },
+      ]}
+    >
+      <View style={styles.container}>
+        <Flex
+          flexDirection={"column"}
+          alignItems={"center"}
+          alignSelf="stretch"
+        >
+          <Flex height={100} justifyContent="center">
+            <BoxedIcon
+              Icon={CloseMedium}
+              backgroundColor={colors.opacityDefault.c05}
+              size={64}
+              variant="circle"
+              borderColor="none"
+              iconSize={32}
+              iconColor={colors.error.c60}
+            />
+          </Flex>
+
+          <Text
+            variant={"h4"}
+            fontWeight="semiBold"
+            textAlign={"center"}
+            numberOfLines={3}
+            mb={2}
+            mt={24}
+          >
+            {"Transaction already validated"}
```

Comment:
Should we put this sentence into translation other than hard code it here in English?